PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azuretools/ijidea/ui/SignInWindow.java

```diff
@@ -131,27 +140,20 @@ public void actionPerformed(ActionEvent e) {
 
         Font labelFont = UIManager.getFont("Label.font");
 
-        interactiveRadioButton.setSelected(true);
-        onInteractiveRadioButton();
+        deviceLoginRadioButton.setSelected(true);
+        refreshAuthControlElements();
 
         init();
     }
 
-    private void onInteractiveRadioButton() {
-        enableAutomatedAuthControls(false);
-    }
-
-    private void onAutomatedRadioButton() {
-        enableAutomatedAuthControls(true);
-    }
-
-    private void enableAutomatedAuthControls(boolean enabled) {
-        interactiveCommentLabel.setEnabled(!enabled);
-        automatedCommentLabel.setEnabled(enabled);
-        authFileLabel.setEnabled(enabled);
-        authFileTextField.setEnabled(enabled);
-        browseButton.setEnabled(enabled);
-        createNewAuthenticationFileButton.setEnabled(enabled);
+    private void refreshAuthControlElements() {
+        interactiveCommentLabel.setEnabled(interactiveRadioButton.isSelected());
```

Comment:
Better not to mess them up. Group the statements by the login methods.

Reply:
Will separate to three methods: `refreshAutomateLoginElements` and `refreshDeviceLoginElements` and `refreshInteractiveLoginElements`.